storage_service/storage_service/settings/base.py

```diff
@@ -258,12 +261,16 @@ def get_env_variable(var_name):
 
 
 # ######## LOGGING CONFIGURATION
-# See: https://docs.djangoproject.com/en/dev/ref/settings/#logging
-# A sample logging configuration. The only tangible logging
-# performed by this configuration is to send an email to
-# the site admins on every HTTP 500 error when DEBUG=False.
 # See http://docs.djangoproject.com/en/dev/topics/logging for
 # more details on how to customize your logging configuration.
+
+# Configure logging manually
+LOGGING_CONFIG = None
+
+# Location of the logging configuration file that we're going to pass to
+# `logging.config.fileConfig` unless it doesn't exist.
+LOGGING_CONFIG_FILE = '/etc/archivematica/storageService.logging.json'
```

Comment:
Instead of hardcoding it, can we use an environment variable that points to the config file, and use /etc/archivematica/storageService.logging.json as the default value (in case the variable is not setted)?

Reply:
That's a good idea. Can we do this in 1.8? https://github.com/artefactual/archivematica/pull/782 has been merged already and it does the same thing.